styles.css

```diff
@@ -0,0 +1,414 @@
+*{
+    box-sizing:border-box;
+}
+
+a{
```

Comment:
Always avoid adding css directly to `html` tags

Reply:
Will take care of it from now onwards.